backend/cfg/dataflow.mli

```diff
@@ -0,0 +1,42 @@
+module type Domain = sig
+  type t
+
+  val equal : t -> t -> bool
```

Comment:
Domain should have an ordering relation "less than or equal", not only "equal", and also "bot".

Reply:
Added.